Modules/_sqlite/connection.c

```diff
@@ -1442,6 +1457,173 @@ pysqlite_connection_iterdump(pysqlite_Connection* self, PyObject* args)
     return retval;
 }
 
+#ifdef HAVE_BACKUP_API
+static PyObject *
+pysqlite_connection_backup(pysqlite_Connection* self, PyObject* args, PyObject* kwds)
+{
+    PyObject* target = NULL;
+    const char* filename;
+    int pages = -1;
+    PyObject* progress = Py_None;
```

Comment:
PEP 7, the ``*`` come before the var name, not after the type. It's ``PyObject *progress``. Same for ``char *name``.

Reply:
I've used the same style as the whole unit, do you think I should follow PEP7 in only new function?
I personally prefer that style, but seemed better to follow the code around... the same reasoning I had with the backup.py (but that's completely new...)

Reply:
Are you sure? :)
```
+    sqlite3 *bckconn;
+    sqlite3_backup *bckhandle;
```
I guess muscle memory is hard to beat...

Your remark is valid. I don't care that much about PEP 7, as long as you stick to one one style. It's really up to you, either obey PEP 7 for new code or stick to the existing style.